stdlib/public/core/FloatingPoint.swift

```diff
@@ -1157,7 +1157,7 @@ public protocol FloatingPoint: SignedNumeric, Strideable, Hashable
 
   /// A Boolean value indicating whether the instance is infinite.
   ///
-  /// Note that `isFinite` and `isInfinite` do not form a dichotomy, because
+  /// Note that `isFinite` and `isInfinite` don't form a dichotomy, because
```

Comment:
"form a dichotomy"??   We should really just shorten this to:

> Note that if `x` is `NaN`, then `x.isFinite` and `x.isInfinite` are both `false`.

Reply:
Changed in df8f551d7f4ba96823416bf8d179aaab7d0cb529, trying an even simpler wording and adding this note under both `isFinite` and `isInfinite`.